docs/concepts/landing.md

```diff
@@ -0,0 +1,20 @@
+---
+id: landing
+title: Legacy concept docs
+description: Legacy category for concepts
+sidebar_label: Concepts
+tags:
+  - introduction
+---
+
+:::info Legacy docs set
+
+The information in this specific sidebar is now largely reflected in the [concept guides](/concepts).
```

Comment:
```suggestion
The information in this specific sidebar (left nav) is now largely reflected in the [concept guides](/concepts).
```
Secondary term added for clarification because "sidebar" has multiple meanings. (I debated expanding the shortened term "nav" but decided in favor of brevity.)